backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlApiFactory.java

```diff
@@ -101,13 +104,44 @@ public static String convertExecutionResultToJson(ExecutionResult executionResul
   /** bit unfortunate that we have to convert from json to map and back */
   static Object transform(String json) throws IOException {
     // benchmark shows this only takes a few ms so not a large performance issue
+    // alternatively, we should change the SQL to result escaped results but that is a nightmare to
```

Comment:
'should' or 'could'

Reply:
what do you think? I think if it becomes slow we 'should'